### DISCUSSION TITLE
Opt-out of suggestions

### DISCUSSION THREAD

**Discussion by lodembeep** [community/community#51340] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Please allow us to opt-out of the suggestions and "For You" features.
These are polluting my space showing me things that i don't care about totally, if i want something i know how to find it.

Same goes for the explore feature, github feels more like facebook for repositories which is waaaay beyond what it is supposed to be.

(now i am seriously thinking about moving to gitlab like many others)

How about showing the discussions and issues i participate to instead ?

**Comment by adlai** [comment]
Until corporate force-pushes its next beta shitdrop, try bookmarking the notifications inbox using your own web browser; the link should work even if you don't have any unread notifications currently.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
